python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -300,6 +305,9 @@ def test_torch_tensor_auto(ray_start_regular, num_gpus):
         ref = compiled_dag.execute(i, shape=shape, dtype=dtype)
         assert ray.get(ref) == (i, shape, dtype)
 
+    # Same reason as comment for teardown in test_torch_tensor_nccl
```

Comment:
```suggestion
    # We need to explicitly teardown because Python will not del right
    # when the reassign happens, so channels will not be properly closed
    # before we open new ones on these actors below.
```

nit: just repeat it for convenience.